vhpidirect/arrays/intvector/vhdlsized/caux.c

```diff
@@ -0,0 +1,24 @@
+#include <assert.h>
+#include <malloc.h>
+#include <stddef.h>
+
+int* allocIntArr(int arrSize){
+  return malloc(arrSize*sizeof(int));
+}
+
+void initIntArr(int* ptr, int arrSize) {
+  for (int i = 0; i < arrSize; i++) {
+    ptr[i] = 11*(i+1);
+  }
+}
+
+void checkAndPrintIntArr(int* ptr, int arrSize) {
+  for (int i = 0; i < arrSize; i++) {
+    assert(ptr[i] == -3 * 11 * (i+1));
+    printf("%d: %d\n", i, ptr[i]);
```

Comment:
I like to printf() before my asserts, So I know which value cause the assert to fail. Really optional, would only help if someone adjusts the VHDL to experiment.

Reply:
Good catch!